src/icons/BakingIcon.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+export default function BakingIcon({ className }) {
```

Comment:
you won't need the `className`, also see line 10

Reply:
Need them for styling!